src/js/disability-benefits/686/containers/IntroductionPage.jsx

```diff
@@ -31,11 +31,34 @@ class IntroductionPage extends React.Component {
           <ol>
             <li className="process-step list-one">
               <div><h5>Prepare</h5></div>
-              <div><h6>To fill out this application, you’ll need your:</h6></div>
+              <div><h6>When you apply, besure to have these on hand:</h6></div>
               <ul>
-                <li>Social Security number (required)</li>
+                <li>Social Security number</li>
+                <li>Current and previous marriage details</li>
+                <li>Unmarried child’s name, Social Security number, date of birth, and school information</li>
               </ul>
-              <p><strong>What if I need help filling out my application?</strong> An accredited representative, like a Veterans Service Officer (VSO), can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a>.</p>
+              <div><h6>If you're claiming a dependent who's over the age of 18 and attending school, you'll also need:</h6></div>
+              <ul>
+                <li>To turn in a completed Request for Approval of School Attendance (VA Form 21-674).</li>
+                <div>
+                  <a href="https://www.vba.va.gov/pubs/forms/VBA-21-674-ARE.pdf">Download VA Form 21-674</a>
+                </div>
+              </ul>
+              <div><h6>What if I need help filling out my application?</h6></div>
+              <p>An accredited representative, like a Veterans Service Officer (VSO), can help you fill out your claim.</p>
+              <div><a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a></div>
+              <br/>
+              <SaveInProgressIntro
```

Comment:
I don't think you need to use SaveInProgressIntro for this. You can use the info version of our AlertBox: https://department-of-veterans-affairs.github.io/design-system/components/detail/alertbox.html